src/Effect/Ref.c

```diff
@@ -0,0 +1,29 @@
+#include <purescript.h>
+
+PURS_FFI_FUNC_2(Effect_Ref_new, val, _, {
+  purs_any_t * ref = purs_new(purs_any_t);
+  return (const purs_any_t *)PURS_ANY_FOREIGN_NEW(ref, val);
```

Comment:
we don't need `ref` here at all and we can just set it to `NULL`. It's not used anywhere else.

Reply:
It works when I set to NULL